doc/sphinx/language/extensions/compil-steps.rst

```diff
@@ -0,0 +1,80 @@
+.. _compil-steps:
```

Comment:
This section, after revision, should be under "Basic notions and conventions", maybe just before [here](https://coq.inria.fr/doc/V8.20.0/refman/language/core/basic.html#essential-vocabulary)